clcache.py

```diff
@@ -1265,16 +1268,20 @@ def postprocessHeaderChangedMiss(
 def postprocessNoManifestMiss(
         cache, objectFile, manifestSection, manifestHash, baseDir, sourceFile, compilerResult, stripIncludes):
     returnCode, compilerOutput, compilerStderr = compilerResult
-    listOfIncludes, compilerOutput = parseIncludesList(compilerOutput, sourceFile, baseDir, stripIncludes)
+    listOfIncludes, compilerOutput = parseIncludesList(compilerOutput, sourceFile, stripIncludes)
 
     manifest = None
     cachekey = None
 
     if returnCode == 0 and os.path.exists(objectFile):
         # Store compile output and manifest
-        manifest = Manifest(listOfIncludes, {})
-        includesContentHash = ManifestsManager.getIncludesContentHashForFiles(
-            [expandBasedirPlaceholder(include, baseDir) for include in listOfIncludes])
+        if baseDir:
+            baseDir = os.path.normcase(baseDir)
```

Comment:
I'll have a look, I tend to agree that normalisation should happen as early as possible.